pkg/services/store/object/dummy/dummy_server.go

```diff
@@ -23,13 +24,14 @@ type ObjectVersionWithBody struct {
 }
 
 type RawObjectWithHistory struct {
-	*object.RawObject `json:"rawObject,omitempty"`
```

Comment:
The embedded structs get really gnarly with JSON encoding, so I just switched to a normal one :_